sp800-63c/sec4_federation.md

```diff
@@ -52,20 +51,20 @@ In this model, a third-party sits in the middle of the transaction and communica
 
  
 
-Brokers can enable simplified technical integrations between the RP and CSP by eliminating the need for multiple point to point integrations. Brokers can also provide business confidentiality and mitigate some of the privacy risks of point to point federation described above by passing the assertions while blinding the participants on either side of the transaction to each other. For example, organizations may not wish to reveal their subscriber lists to each other and blinding prevents the capability for CSPs or RPs to track and profile subscribers. However, this model transfers this capability to the broker.  
+Brokers can enable simplified technical integrations between the RP and CSP by eliminating the need for multiple point to point integrations, which can be onerous for protocols which do not support [dynamic registration](#dynamic-registration). If implemented in very specific ways, brokers can also provide some business confidentiality and transfer some of the privacy risks of point to point federation described above by passing the assertions while blinding the participants on either side of the transaction to each other. For example, organizations may not wish to reveal their subscriber lists to each other and some implementations of blinding technology somewhat prevent the capability for CSPs or RPs to track and profile subscribers. However, this model transfers this capability to the broker.  While the use of brokers can give the appearance of complete privacy preservation, in many instances the implementation of broker technology still allows the tracking of subscribers, CSPs, and RPs in different ways.  
```

Comment:
For example sentence - remove somewhat, add within the authentication transaction.